_posts/2020-07-21-cube-functionality-to-kartothek.markdown

```diff
@@ -0,0 +1,409 @@
+7        22                   NaN  Santiago      CL 2020-07-02        NaN        NaN
+ 
+```
+
+## Remove and Delete Operations
+
+You can **remove** entire partitions from the cube using the remove operation.
+
+```python
+>>> from kartothek.io.eager_cube import remove_partitions
+>>> datasets_after_removal = remove_partitions(
+...     cube=cube,
+...     store=store,
+...     ktk_cube_dataset_ids=["latlong"],
+...     conditions=(C("country") == "UK"),
+... )
+>>> query_cube(
+...     cube=cube,
+...     store=store,
+... )[0]
+   avg_temp  avg_temp_country_min      city country        day   latitude  longitude
+0         8                   6.0   Dresden      DE 2020-07-01  51.050407  13.737262
+1         4                   4.0   Dresden      DE 2020-07-02  51.050407  13.737262
+2         6                   6.0   Hamburg      DE 2020-07-01  53.551086   9.993682
+3         5                   4.0   Hamburg      DE 2020-07-02  53.551086   9.993682
+4         6                   6.0    London      UK 2020-07-01        NaN        NaN
+5         8                   4.0    London      UK 2020-07-02        NaN        NaN
+6        20                   NaN  Santiago      CL 2020-07-01        NaN        NaN
+7        22                   NaN  Santiago      CL 2020-07-02        NaN        NaN 
+```
+
+You can also **delete** entire datasets (or the entire cube).
+
+```python
+>>> from kartothek.io.eager_cube import delete_cube
+>>> datasets_still_in_cube = delete_cube(
+...     cube=cube,
+...     store=store,
+...     datasets=["transformed"],
+... )
+>>> query_cube(
+...     cube=cube,
+...     store=store,
+... )[0]
+   avg_temp      city country        day   latitude  longitude
+0         8   Dresden      DE 2020-07-01  51.050407  13.737262
+1         4   Dresden      DE 2020-07-02  51.050407  13.737262
+2         6   Hamburg      DE 2020-07-01  53.551086   9.993682
+3         5   Hamburg      DE 2020-07-02  53.551086   9.993682
+4         6    London      UK 2020-07-01        NaN        NaN
+5         8    London      UK 2020-07-02        NaN        NaN
+6        20  Santiago      CL 2020-07-01        NaN        NaN
+7        22  Santiago      CL 2020-07-02        NaN        NaN
+```
+
+## Cube Features in Kartothek
+
+*	**Multiple-datasets**: When mapping multiple parts (tables or datasets) to Kartothek, using multiple datasets allow users to copy, backup and delete them separately.
+ Index structures are bound to datasets. This was not possible with the existing multi-table (within a single dataset) feature present in kartothek.
+ We intend to phase out the multi-table single dataset functionality soon.
+
+*	**Seed-Based Join System / Partition-alignment**: When data is stored in multiple parts (tables or datasets), the question is how to expose it to the user during read operations.
+ Seed-based join marks a single part as seed which provides seed dataset in the cube, all other parts are just additional columns.
```

Comment:
```suggestion
 Seed-based join marks a single part as ground-truth which provides the row set present in the cube, all other parts are just additional columns.
```

You're explaining "seed" with "seed" otherwise.